includes/rules.php

```diff
@@ -74,6 +74,14 @@
 			'<code>&lt;h2&gt;</code>'
 		),
 	),
+	array(
+		'title'     => esc_html__( 'Empty Paragraph Tags', 'accessibility-checker' ),
```

Comment:
@pattonwebz please make title singular - "Empty Paragraph Tag" - this will match how we have named other rules.